R/algorithm.R

```diff
@@ -161,6 +161,24 @@ algorithm <- function() {
       title = "Metformin purchases that aren't potentially for the treatment of PCOS",
       logic = "NOT (koen == 2 AND atc =~ '^A10BA02$' AND ((date - foed_dato) < years(40) OR indication_code %in% c('0000092', '0000276', '0000781')))",
       comments = "Woman is defined as 2 in `koen`."
+    ),
+    is_any_t1d_primary_diagnosis = list(
+      register = NA,
+      title = "Any primary diagnosis for type 1 diabetes",
+      logic = "(n_t1d_endocrinology + n_t1d_medical) >= 1",
+      comments = "This is used to classify type 1 diabetes."
+    ),
+    is_endocrinology_dx = list(
+      register = NA,
+      title = "Diagnosis of diabetes from an endocrinology department",
+      logic = "n_t1d_is_endocrinology_dept AND (is_t1d_code OR is_t2d_code)",
+      comments = "This is used to classify type 1 diabetes."
+    ),
+    t1d = list(
```

Comment:
```suggestion
    is_t1d = list( 
```
To fit the naming scheme?

Reply:
`t1d` column will be exposed to the user. So I don't know how we want to name it. Maybe `has_t1d`? @Aastedet what do you think?

Reply:
I like `has_t1d` better, and maybe `has_*` in general, actually, instead of `is_`